app/request_handler.py

```diff
@@ -29,6 +29,22 @@
 firebase = pyrebase.initialize_app(FIREBASE_CONFIG)
 db = firebase.database()
 
+def writeYelpRecords(yelpVenues):
+    record = {}
+
+    for biz in yelpVenues:
+        key   = representation.createKey(biz)
+        key   = key.encode('utf-8').strip()
```

Comment:
No -- this was based off existing code. I also don't like `=` alignment, so I'll just remove it.